src/components/FilterPanel/FilterCheckbox/_mixins.scss

```diff
@@ -0,0 +1,39 @@
+////
+/// Filter checkbox mixins.
+/// @group filter-panel
+/// @copyright IBM Security 2020
+////
+
+@import '../../../globals/index';
+
+@mixin filter-checkbox {
+  &__wrapper {
+    @include carbon--type-style($name: body-short-01);
+    display: flex;
+    max-width: 100%;
+  }
+
+  &__count {
+    @include carbon--type-style($name: caption-01);
+
+    // Add visual parentheses.
+    &::before {
+      content: '(';
+    }
+
+    &::after {
+      content: ')';
+    }
```

Comment:
What about this makes you nervous? The parentheses are visual flourishes that don't carry any content meaning.

Reply:
This approach is actually how GH adds visual flourishes as well: 
![image](https://user-images.githubusercontent.com/5973294/72557998-78eff480-3867-11ea-8c88-e5f9595280ea.png)

Reply:
The GH diff situation is more extreme to me, because that's even less of a visual flourish... there doesn't seem to be any indication that a given line is being added or reviewed, except through visual indicators with colors + the `:before` content? I tried to see what it reads as in VoiceOver, and it just reads the line minus +/- or any context about adding/removing the line.

With the count parentheses, VoiceOver reads as "Filter label 10" or whatever, where 10 is just part of the line... that seems kinda weird to me and not as helpful as it could be. It would be better if there was a pause and then "10 items" or something of that nature. You could add screen-reader only text inside the parenthesis, after the count number, to achieve this? 🤔

Reply:
Okay, I've added a `countLabel` prop that I think will with this.